data.js

```diff
@@ -103,16 +103,36 @@ exports.data = [
             label: "greenpeace",
         },
         description: {
-            content: "Greenpeace is another big player. They also do work on a big scale as you can see on the Income they have to put them into their projects. Greenpeace has a lot of Actions going on.",
-            updatedBy: "Andre", updatedAt: "2019-08-18"
+            content: "Greenpeace is another big player, working in independent national and regional organizations. They do work on a big scale as you can see on the income and the variety of actions.",
```

Comment:
Let's remove "another", it's confusing if Greenpeace is the only suggestion or on top of the list.

Reply:
Brevity: `They do work on a big scale in terms of funding, staff and variety of actions`.

Reply:
changed the whole wording